main.nf

```diff
@@ -8,18 +8,20 @@ include { humann; humann_regroup; humann_rename } from './processes/humann.nf'
 
 
 workflow {
-    
-    read_pairs_ch = Channel
-        .fromFilePairs("$params.readsdir/$params.filepattern", size: 2)
+    read_ch = Channel
+        .fromPath("$params.readsdir/$params.filepattern")
```

Comment:
Need to revert this - the `fromPath` channel just returns a list of files, and doesn't group them

Can also pass vectors of patterns, and can probably incorporate into the yamls, eg

```
filepatterns:
  - "*.fastq"
  - "*_L00{1,2,3,4}_R{1,2}_001.fastq.gz"
```

or something